File: Modules/ServerTypes/Layanan/StatusEnum.ts
import { Decorators } from "@serenity-is/corelib";

export enum StatusEnum {
    Diproses = 1,
    Disetujui = 2,
    Ditolak = 3
}
Decorators.registerEnumType(StatusEnum, 'PengaduanMasyarakat.Layanan.StatusEnum', 'Layanan.StatusEnum');

File: Modules/Layanan/Pengaduan/StatusEnum.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace PengaduanMasyarakat.Layanan
{
    [EnumKey("Layanan.StatusEnum")]
    public enum StatusEnum {
        [Description("Diproses")]
        Diproses = 1,
        [Description("Disetujui")]
        Disetujui = 2,
        [Description("Ditolak")]
        Ditolak = 3,
    }
}